src/components/Login.vue

```diff
@@ -49,6 +53,15 @@
 export default {
   data() {
     return {
+      valid: false,
+      emailRules: [
+        v => !!v || 'Email is required',
+        v => /.+@.+/.test(v) || 'Email must be valid',
+      ],
+      passwordRules: [
+        v => !!v || 'Password is required',
+        v => v.length >= 8 || 'Password must be greater than 8 characters',
```

Comment:
This rule is not required for logging in

Reply:
U could just add a common required field rather than have 2. (duplication of code).
Make a common validation/rules object with each of the rules as part of an array